performance/benchmarks/benchmark_spherical_representations.py

```diff
@@ -42,7 +42,7 @@
 from rascal.representations import SphericalExpansion
 
 # Benchmark hyperparameters
-NB_ITERATIONS_PER_REPRESENTATION = 20
+NB_ITERATIONS_PER_REPRESENTATION = 100
```

Comment:
No, but using 100 do not takes too much time, so it should be worth it anyway :smiley:

Reply:
I just checked the circleci running time, and for the clang-9-debug one the benchmark takes 5 mins. 
Quick solution: Change this to 1 for now
Solution: Make NB_ITERATIONS_PER_REPRESENTATION a optional input parameter for the python benchmark, then set this one to 1 for the circleci (as it is done for the google benchmarks on the c++ side)

I am okay to with quick solutions for this PR, and add later the real solution. I just don't want that other people have long circleci times because of the benchmarks.

Reply:
Sounds good, I'll revert it to 20 and check benchmark runtime.

I just found out about https://pypi.org/project/pytest-benchmark/, which might be interesting here.

Reply:
added it to issue #218

Reply:
Reverted to 20. Benchmark time is still high-ish with clang-debug, but it seems this was already the case before.

Reply:
okay I see, added issue to #218